web-ui/src/main/resources/catalog/templates/top-toolbar.html

```diff
@@ -197,7 +197,7 @@
         </ul>
       </li>
       <li class="dropdown dropdown-hover open signin-dropdown"
-        data-ng-if="!authenticated && service !== 'catalog.signin' && service !== 'new.account' && !shibbolethEnabled">
+        data-ng-if="!authenticated && service !== 'catalog.signin' && service !== 'new.account' && !shibbolethHideLogin">
```

Comment:
Same as previous, would not be better to check both `shibbolethEnabled` and `shibbolethHideLogin`?